packages/analytics-browser/src/browser-client.ts

```diff
@@ -186,7 +186,6 @@ export class AmplitudeBrowser extends AmplitudeCore implements BrowserClient {
 
     this.config.sessionId = sessionId;
     this.config.lastEventTime = undefined;
-    this.config.pageCounter = 0;
```

Comment:
So what is the expectation when someone calls setSessionId. Is it ok that the page counter will not go to 0 in that case?